filebeat/module/elasticsearch/audit/test/test-audit-docker.log

```diff
@@ -1,3 +1,2 @@
 {"type": "audit", "timestamp":"2019-06-11T15:03:32,102+0000", "node.id":"Xaq2BFVcQ1OhyMrjL8gNOg", "event.type":"rest", "event.action":"anonymous_access_denied", "origin.type":"rest", "origin.address":"172.17.0.1:40380", "url.path":"/", "request.method":"GET", "request.id":"pkduyMB5Tly6xgmkYbZi-A"}
-{"type": "server", "timestamp": "2019-06-11T15:03:32,777+0000", "level": "INFO", "component": "o.e.x.s.a.AuthenticationService", "cluster.name": "docker-cluster", "node.name": "dff7befc418f", "cluster.uuid": "xEiKc6ipRiyzU8_8czXrJw", "node.id": "Xaq2BFVcQ1OhyMrjL8gNOg",  "message": "Authentication of [elastic] was terminated by realm [reserved] - failed to authenticate user [elastic]"  }
```

Comment:
in fact this is a real sample. Docker logs are all emitted into the same std out stream, so audit logs will be interleaved.
I would leave the server log line here, but remove the json from the expected file.